### PR TITLE
Create merchant repo

### PR DESCRIPTION
**Pull request by jbranham1**
Created merchant repo that builds the merchant records.
Fixed simplecov to actually check our files.

Next Steps: Refactor to create module for repo/parameters

### CODE REVIEW

File: lib/merchant_repository.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require_relative "merchant"
```
[the remark this answers is not in the document]
Comment (jbranham1) [reply lib/merchant_repository.rb]: Resolved single quotes strings

File: test/item_test.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
-require 'minitest/autorun'
-require 'minitest/pride'
+require './test/test_helper'
 require 'bigdecimal'
 require './lib/item'
 
+
```
Comment (jbranham1) [inline test/item_test.rb]: Resolved

**Comment by jbranham1** [comment]
@aschwartz1 This is ready for you to review!

File: test/merchant_repository_test.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+require './test/test_helper'
+require 'csv'
+require './lib/merchant_repository'
+require 'pry'
```
Comment (aschwartz1) [inline test/merchant_repository_test.rb]: To clean up the requires, could you remove the pry reference?

File: test/merchant_repository_test.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  def test_it_exists
+    assert_instance_of MerchantRepository, @repo
+  end
+
+  def test_build_merchants
+    @repo.build_merchants
+    assert_equal 2, @repo.merchants.count
+  end
+
+  def test_merchants
```
Comment (aschwartz1) [inline test/merchant_repository_test.rb]: @Jenny `test_merchants` and `test_build_merchants` are the same -- could they be consolidated, or is there something separate to test in each method which could be more explicit?

Comment (aschwartz1) [reply test/merchant_repository_test.rb]: (Was SimpleCov needing both to get to 100%?)

**Review by aschwartz1 — DISMISSED** [review]
Great job.

I commented a couple lines of code. Please let me know your thoughts!

File: test/merchant_repository_test.rb (hunk 3 above)
Comment (jbranham1) [reply test/merchant_repository_test.rb]: Good catch! Removed pry

File: test/merchant_repository_test.rb (hunk 4 above)
Comment (jbranham1) [reply test/merchant_repository_test.rb]: Was able to remove the merchants test and it still ran as expected with 100% on SimpleCov

**Comment by jbranham1** [comment]
@aschwartz1 Ready for you again!

**Comment by aschwartz1** [comment]
Great!

**Review by aschwartz1 — APPROVED** [review]

**Pull request merged**